### ISSUE TITLE
Mysterious durability bug, likely in the implementation

### ISSUE THREAD

**Issue by erights** [Agoric/Agoric#5628]
The PR https://github.com/Agoric/agoric-sdk/pull/5627 is a trivial change from another PR https://github.com/Agoric/agoric-sdk/pull/5283 that is in a much more working state (only one CI failure due to a bug in otc-desk https://github.com/Agoric/dapp-otc/pull/38 ). However, #5627 has lots of CI errors like https://github.com/Agoric/agoric-sdk/runs/6959350964?check_suite_focus=true#step:12:15

```
TypeError#1: Cannot set property 'prev' of undefined
  at Object.refresh (packages/SwingSet/src/liveslots/virtualObjectManager.js:121:22)
  at ensureState (packages/SwingSet/src/liveslots/virtualObjectManager.js:630:17)
  at Object.get (packages/SwingSet/src/liveslots/virtualObjectManager.js:643:13)
  at deposit (.../ertp/src/purse.js:78:[15](https://github.com/Agoric/agoric-sdk/runs/6959350964?check_suite_focus=true#step:12:16))
  at Alleged: baseball cards purse.method [as deposit] (packages/SwingSet/src/liveslots/virtualObjectManager.js:478:43)
```
which is at a level of abstraction far below the PR change.

**Comment by erights** [comment]
@FUDCo reports that I am reusing a kindHandle. He's also working a fix at his end to detect and complain when a kindHandle is reused, which will help us diagnose these early. I'm waiting on that to fix this bug.
